Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lights/SimpleSpotLight.azsli

```diff
@@ -59,6 +60,15 @@ void ApplySimpleSpotLight(ViewSrg::SimpleSpotLight light, Surface surface, inout
         }
 #endif
 
+        // apply gobo texture
+        if (light.m_goboTexIndex < ViewSrg::MaxGoboTextureCount)
+        {
+            float4 positionInView = mul(light.m_viewProjectionMatrix, float4(surface.position, 1.0));
+            positionInView /= positionInView.w;
+            float2 goboUv = float2((positionInView.x + 1.0)/2.0, (positionInView.y + 1.0)/2.0);
+            lightIntensity *= ViewSrg::m_goboTextures[light.m_goboTexIndex].Sample(PassSrg::LinearSampler, goboUv);
```

Comment:
would be kinda cool if we supported animating the gobo UV with an offset over time.

Reply:
There could be many different ways to animated the UV. I wouldn't complicate the shader (extra calculations) unless the requirement is clear.